package.json

```diff
@@ -55,22 +63,24 @@
   "keywords": [
     "scratch"
   ],
+  "husky": {
```

Comment:
I would recommend to use `husky` (https://github.com/typicode/husky) with `lint-staged` as `lint-staged` document explains (https://github.com/okonet/lint-staged#examples).

Reply:
`husky` set up git hooks automatically.